.github/workflows/backend.yml

```diff
@@ -109,6 +109,120 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-in-control-mode:
+    if: needs.files-changed.outputs.backend == 'true'
+    needs: files-changed
+    name: backend test in control mode
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        instance: [0]
+        pg-version: ['9.6']
+
+    env:
+      MATRIX_INSTANCE_TOTAL: 1
+      MIGRATIONS_TEST_MIGRATE: 1
+      SENTRY_SILO_MODE: CONTROL
```

Comment:
seems like this could just be an input rather than an entire job being copy pasted